### PR TITLE
copied atreus astar to mirrored, set PCBDOWN flag

### LINKED ISSUE #5952: Atreus mirrored layout support in configurator
## Feature Request Type

- [ ] Core functionality
- [ ] Add-on hardware support (eg. audio, RGB, OLED screen, etc.)
- [x] Alteration (enhancement/optimization) of existing feature(s)
- [ ] New behavior

## Description

Add support for mirrored layout (A-star) in the configurator.  

From what I can tell, this would entail either updating the configurator to support passing flags (PCBDOWN=yes) to the worker who compiles the firmware or, perhaps more easily, adding a mirrored layout (eg LAYOUT_AStar) to the Atreus's [atreus.h](https://github.com/qmk/qmk_firmware/blob/master/keyboards/atreus/atreus.h).

### PR DESCRIPTION
**Pull request by RagingRoosevelt**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR
* Fixes #5952
* Fixes qmk/qmk_configurator#537

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/atreus/astar_mirrored/astar_mirrored.c [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "atreus_mirrored.h"
```
Comment (noroadsleft) [inline keyboards/atreus/astar_mirrored/astar_mirrored.c]:
```suggestion
#include "astar_mirrored.h"
```

File: keyboards/atreus/astar_mirrored/config.h [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+*/
+#define PDBDOWN 1
```
Comment (zvecr) [inline keyboards/atreus/astar_mirrored/config.h]: this is always going to be true, so this block should just be
```c
#define MATRIX_ROW_PINS { D0, D1, D3, D2 }
#define MATRIX_COL_PINS { B7, D6, F7, F6, B6, D4, E6, B4, B5, C6, D7 }
```

File: keyboards/atreus/astar_mirrored/astar_mirrored.c (hunk 1 above)
Comment (RagingRoosevelt) [reply keyboards/atreus/astar_mirrored/astar_mirrored.c]: Thanks, not sure how I missed that.

File: keyboards/atreus/astar_mirrored/config.h (hunk 2 above)
Comment (RagingRoosevelt) [reply keyboards/atreus/astar_mirrored/config.h]: I made the adjustment.  I was thinking just using the define would keep the code as similar to existing as possible.  This makes sense too, though.

Comment (zvecr) [reply keyboards/atreus/astar_mirrored/config.h]: Hopefully, we can deprecate the old PCB_DOWN flag in the next breaking changes window. And then the non pcb down version will look like this one.

**Comment by RagingRoosevelt** [comment]
I have used the new layout profile to flash my mirrored astar atreus with a layout and it worked flawlessly.  Thank you for helping me get the code in shape.  I really appreciate how welcoming the QMK community has been.

-sent from my astar atreus

**Pull request merged**